Makefile

```diff
@@ -98,7 +89,6 @@ rebuild-dependencies.tsv: godeps
 # PPA includes the required mongodb-server binaries.
 install-dependencies:
 	@echo Adding juju PPAs for golang and juju
-	@sudo apt-add-repository --yes ppa:juju/golang
```

Comment:
given 'make check' is how some of the CI jobs work, are we sure this won't actually cause problems on non-developer machines?